### ISSUE TITLE
Front controller reached 100 router match iterations

### ISSUE THREAD

**Issue by davidwindell** [magefan/module-blog#244]
We're seeing repeated instances of this error in our logs, this is on the latest release `2.8.9.1` running on Magento 2.3.1.

```json
{
	"0": "Front controller reached 100 router match iterations",
	"1": "<pre>#1 Magento\\Framework\\App\\FrontController\\Interceptor->___callParent() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:138]\n#2 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/module-store\/App\/FrontController\/Plugin\/RequestPreprocessor.php:94]\n#3 Magento\\Store\\App\\FrontController\\Plugin\\RequestPreprocessor->aroundDispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:135]\n#4 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/outeredge\/magento-base-module\/Plugin\/DbStatusValidator.php:55]\n#5 OuterEdge\\Base\\Plugin\\DbStatusValidator->aroundDispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:135]\n#6 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/module-page-cache\/Model\/App\/FrontController\/BuiltinPlugin.php:69]\n#7 Magento\\PageCache\\Model\\App\\FrontController\\BuiltinPlugin->aroundDispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:135]\n#8 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:153]\n#9 Magento\\Framework\\App\\FrontController\\Interceptor->___callPlugins() called at [generated\/code\/Magento\/Framework\/App\/FrontController\/Interceptor.php:26]\n#10 Magento\\Framework\\App\\FrontController\\Interceptor->dispatch() called at [vendor\/magento\/framework\/App\/Http.php:136]\n#11 Magento\\Framework\\App\\Http->launch() called at [vendor\/magento\/framework\/App\/Bootstrap.php:258]\n#12 Magento\\Framework\\App\\Bootstrap->run() called at [pub\/index.php:40]\n<\/pre>",
	"url": "\/blog\/rss\/index\/store_id\/1\/",
	"script_name": "\/index.php"
}
```

**Comment by magefan** [comment]
This issue already has been fixed https://github.com/magefan/module-blog/commit/441b3b5ce7d7b8eee336a559ba2a0ba5fe0d4d78 and will be included in next release 2.8.10

**Comment by davidwindell** [comment]
Awesome!
